assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/buttons/useSubmitButtonProps.tsx

```diff
@@ -10,7 +12,7 @@ interface Props extends Prices {
 }
 
 const useSubmitButtonProps = ({ onCloseModal, prices }: Props): ButtonProps => {
-	const disabled = prices.length && prices.some(({ amount }) => anyPass([isNil, isEmpty])(amount));
+	const isDisabled = prices.length && prices.some(({ amount }) => anyPass([isNil, isEmpty])(amount));
```

Comment:
nice, `isDisabled` expresses that it is a boolean :+1:

Reply:
agreed